tutorials/Image_And_Icons_Manipulations/README.md

```diff
@@ -311,3 +311,12 @@ fun main() = application {
 ```
 
 <img alt="Tray icon" src="tray_icon.png" height="479" />
+
+## Loading image resources in commonMain
+Currently, Compose Multiplatform doesn't support loading resources from commonMain sourceSet. You can track this feature [here](https://github.com/JetBrains/compose-jb/issues/1255).
+
+Until then, there is a workaround for loading resources from `commonMain/resources` folder:

```

Comment:
Links are OK, but maybe a step-by-step tutorial with code snippets would be more clear